atomic_reactor/constants.py

```diff
@@ -163,6 +163,10 @@
 
 KOJI_BTYPE_IMAGE = 'image'
 KOJI_BTYPE_OPERATOR_MANIFESTS = 'operator-manifests'
+KOJI_BTYPE_CONTAINER_FIRST = 'container-first'
```

Comment:
Would anyone find this btype name confusing for holding the remote sources archives?

Reply:
We've talked about other names for this:
* upstream-sources (implies a downstream)
* container-sources
* ...

What about 'remote-sources', for consistency with what we already call the tarball?

Reply:
Thinking about it, it seems to be the best option here.

2 files will be presented with this type for now: The archive with the remote sources and some metadata with the information used to fetch the remote sources.